runbotics/runbotics-orchestrator-ui/src/layouts/MainLayout/TopBar/LangSwitcher/LangSwitcher.styled.ts

```diff
@@ -1,9 +1,23 @@
-import { FormControl } from '@mui/material';
+import { FormControl, MenuItem } from '@mui/material';
 import styled from 'styled-components';
 
 export const LangFormControl = styled(FormControl)`
     &[class*="MuiFormControl"] {
```

Comment:
z tego co mi wiadomo od MUI 5 używamy albo sx na elemencie albo styled components ale nie odwołując się do klasy wewnątrz. Czy jest opcja by to przerefactorować by nie było &[class*="..."]?

Reply:
problem polega na tym, że MUI ma zagnieżdżone klasy i musiałbym użyć important żeby nadpisać style MUI

Reply:
No dobra, poza tym wygląda spoko